deploy/prometheus/100-ebs-iops-burstbalance.PrometheusRule.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
     prometheus: sre-ebs-iops-burstbalance
     role: alert-rules
   name: sre-ebs-iops-burstbalance
-  namespace: openshift-monitoring
```

Comment:
This was made in the wrong file. While I will be making this change, I want to make it in a separate changeset and not for #3.

Reply:
ok